GameServerApp/Program.cs

```diff
@@ -15,44 +17,97 @@ internal class Program
 
         private static void Main(string[] args)
         {
-            var options = ArgsOptions.Parse(args);
+            _logger = LoggerProvider.GetLogger();
+
+            var parsedArgs = ArgsOptions.Parse(args);
+            var settingsPath = Path.Combine(Directory.GetCurrentDirectory(), "Settings");
 
-            var configJson = options.ConfigJson;
-            if (string.IsNullOrEmpty(configJson))
+            if (string.IsNullOrEmpty(parsedArgs.GameInfoJson))
             {
-                configJson = File.ReadAllText(options.ConfigPath);
+                if (File.Exists(parsedArgs.GameInfoJsonPath))
+                {
+                    parsedArgs.GameInfoJson = File.ReadAllText(parsedArgs.GameInfoJsonPath);
+                }
+                else
+                {
+                    try
+                    {
+                        Directory.CreateDirectory(settingsPath);
```

Comment:
what's up with the odd spacing?

Reply:
its the normal spacing vs applies

Reply:
I mean the newlines under this?

Reply:
oh hm i just thinks its more readable then writing everything under eachother without a single space